src/data/calendar.js

```diff
@@ -0,0 +1,136 @@
+/* global getBoxingDays, getFrenchBankHolidays */
+(function(exports) {
+  "use strict";
+
+  function guessTypeFromComment(comment) {
+    var KNOWN_TYPES = ['RTT', 'CP', 'CS'];
```

Comment:
Actually CSS also means Congés Sans Solde, you can also have JRTT.
I don't think it is a problem to have false positive because you will be able to change it yourself.
However if you go to CSS Conf Mozilla will pay for you so it won't be a PTO.

If we want this comment handling to be perfect, then we should not try to guess the thing.